### PR TITLE
Make TS-check more strict

### LINKED ISSUE #65: Make TS-check more strict

### PR DESCRIPTION
**Pull request by react-developer-ssa-group**
[Make TS-check more strict](https://github.com/ssagroup/ui-kit/issues/65)
- added detailed typing

Closes #65

### CODE REVIEW

File: packages/core/src/components/Wrapper/Wrapper.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import styled from '@emotion/styled';
+import { CommonProps } from '../..';
```
Comment (arthur-kashaiev-ssa-group) [inline packages/core/src/components/Wrapper/Wrapper.tsx:2]: I suggest using another way of import (throughout the project): 
`import { CommonProps } from '@types/emotion';`

I'm unsure whether the `'../..'` will work in all cases.

**Review by arthur-kashaiev-ssa-group — CHANGES_REQUESTED** [review]
Looks good to me. I've just left one small comment.

**Comment by arthur-kashaiev-ssa-group** [comment]
@react-developer-ssa-group This is the commit you need to make the Button types work: 
https://github.com/ssagroup/ui-kit/commit/17340dee2a129070a1c6f3549e248fc4c2b94fef

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 8  | ❗       |
    


<details>
<summary>❗ storybook/components-card--item-card__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w390px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-action__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w1440px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w1920px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>



<details>
<summary>❗ storybook/components-card--item-card-avatar__[w900px] (click to expand)</summary>

| Before | After |
| --- | --- |
|  |  |
</details>

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 2  | ❗       |
    


*❗ custom/widgets-heart-rate--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-heart-rate--hint-opened__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 0  | ❗       |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*❗ custom/widgets-heart-rate--hint-opened__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-heart-rate--hint-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-heart-rate--hint-opened__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-heart-rate--hint-opened__[w900px]*: 


| Before | After |
| --- | --- |
|  |  |

**Review by arthur-kashaiev-ssa-group — APPROVED** [review]

**Pull request merged**